oaknational/Oak-Web-Application
# fix: kebab-case-to-camel-case-line-svg

## Description

- Fix case in svg

I'm surprised none of our tests picked this up.

## How to test

1. Go to footer
2. Line should look the same
3. Check in mobile view
4. Line should look the same

Checking in Percy.


## Checklist
